src/scss/layouts/__layouts-dir.scss

```
@@ -0,0 +1 @@
+@import './layout-home';
```

Newline needed (Resolve when fixed)
```suggestion
@import './layout-home';

```